packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -303,6 +303,13 @@ class TextSelectionOverlay {
     );
   }
 
+  /// Controls the fade-in and fade-out animations for the toolbar and handles.
+  @Deprecated(
+    'Use `SelectionOverlay.fadeDuration` instead. '
+    'This feature was deprecated after 2.12.0-4.1.pre.'
```

Comment:
You have an analyzer failure about this deprecated format.  Maybe it's just a missing trailing comma here?

Reply:
yes